File: FrontEnd/frontend/src/types/ProductoDTO.ts
export interface ProductoDTO {
  id: number;
  nombre: string;
  descripcion: string;
  precio: number;
  stock: number;
  categoriaNombre: string;
  categoriaId: number;
  eliminado: boolean;

  stockStatus: string;
  precioFormateado: string;
}

File: BackEnd/patroncqrs/src/main/java/com/cqrs/patroncqrs/query/dto/ProductoDTO.java
package com.cqrs.patroncqrs.query.dto;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ProductoDTO {

    private Long id;
    private String nombre;
    private String descripcion;
    private Double precio;
    private Integer stock;
    private String categoriaNombre;
    private Long categoriaId;
    private Boolean eliminado;

    private String stockStatus;
    private String precioFormateado;
}
